IntelliTect.TestTools.Console/ConsoleAssert.cs

```diff
@@ -24,7 +23,9 @@ public static class ConsoleAssert
         /// including both input and output</param>
         /// <param name="action">Method to be run</param>
         /// <param name="normalizeLineEndings">Whether differences in line ending styles should be ignored.</param>
-        public static string Expect(string expected, Action action, bool normalizeLineEndings = true)
+        /// <param name="stripVT100">Whether VT100 color code characters should be ignored.</param>
+        public static string Expect(string expected, Action action, bool normalizeLineEndings = true, 
+            bool stripVT100 = true)
```

Comment:
Given we now have normalizeLineEndings and stripVT100, I suspect we should use flags instead of Booleans?  This improves the readability of the calling code in the non-default scenarios as well.

Reply:
I was trying to add this option without breaking changes to prior versions. I agree some kind of flag would make more sense, but I think that would require changes to the signature and thus breaking changes everywhere?

Reply:
I'm not sure what that would be true necessarily.
1. We can remove the default from the `normalizeLineEndings` parameter.
2. Add the `NormalizeOptionsFlag` (not certain on the name)? and default it to `NormalizeLineEndings` on and `StripVT100` on.
3. In the method with no options (either Boolean or flags), invoke the method with the `NormalizeOptionsFlag` parameter.

Reply:
I talked to Kevin. I'll provide a flags overload, that allows the stripVT100 option